ui/app/styles/core/checkbox-and-radio.scss

```diff
@@ -5,8 +5,6 @@
 
 // This file defines the styles for .checkbox, .radio and .b-checkboxes. The prefix "b" comes from Bulma.
 
-@import '../sass-svg-uri/svg-uri';
```

Comment:
from @meirish:
> The `css` in-repo addon was doing some build time things that aren't really supported in Embroider. I looked at adding this to the Sass options like the HDS CSS, but then there was an error about division (see [this issue](https://github.com/waldemarfm/sass-svg-uri/issues/4)). I updated to the latest for recent Sass compatibility, but since the include is now `_index.scss` the only way to use it was to add `node_modules` to the includePath. That was slow. Not just a little slow, it was a lot slow. Slooooooooow. Rebuilds took 27s.
> 
> So. I decided it was only used in one place - I copied the compiled output and dropped it in that one place and then removed the `css` in-repo addon and the `sass-svg-uri` dependency.